Packs/Code42/Integrations/Code42/integration-Code42.yml

```diff
@@ -0,0 +1,1904 @@
+category: Endpoint
```

Comment:
We don't need to commit this file

Reply:
Shall I remove this file from the commit?